_docs/development/index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Overview
+order: 001
```

Comment:
No, it's not with GitHub Pages. That's what I wanted to do, but I'm afraid there's no way to build hierarchical data in Liquid templates (because [no type exists](https://shopify.github.io/liquid/basics/types/)) without plugins.